src/Array/Array.ts

```diff
@@ -22,16 +21,17 @@ import { NonEmptyArray } from "./NonEmptyArray"
  * @group Filtering
  */
 export function filter<A, B extends A>(
-    refinement: Refinement<A, B>
-): (as: readonly A[]) => readonly B[]
+    refinement: Refinement<A, B>,
+): (as: readonly A[]) => readonly B[];
 export function filter<A>(
-    predicate: Predicate<A>
-): <B extends A>(bs: readonly B[]) => readonly B[]
-export function filter<A>(predicate: Predicate<A>): (as: readonly A[]) => readonly A[]
+    predicate: Predicate<A>,
+): <B extends A>(bs: readonly B[]) => readonly B[];
+export function filter<A>(
+    predicate: Predicate<A>,
+): (as: readonly A[]) => readonly A[];
 export function filter<A>(f: Predicate<A>) {
-    return <B extends A>(as: readonly B[]) => as.filter(f)
+    return <B extends A>(as: readonly B[]) => as.filter(f);
```

Comment:
I'm assuming adding semicolons back wasn't intentional?

Reply:
Woops good catch 👍 fixed the Biome config